.github/workflows/nodejs-windows.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     strategy:
       max-parallel: 3
       matrix:
-        node_version: [12.x, 14.x, 16.x, 17.x, 18.x, 20.x]
+        node_version: [10.x, 12.x, 14.x, 16.x, 18.x, 20.x]
```

Comment:
We recommend LTS versions  
I believe it is ok to add this support for sometime to test in CI for any backward compatibility testing.

Reply:
I didn't get it, these are nodejs lts versions

Reply:
do you mean to drop support for end-of-life nodejs LTS version?

https://github.com/nodejs/release#release-schedule

Reply:
This make things much easier